ecosystem/node-checker/src/configuration/types.rs

```diff
@@ -215,6 +214,15 @@ impl NodeAddress {
         url.set_port(Some(self.api_port)).unwrap();
         url
     }
+
+    pub fn get_api_client(&self) -> AptosRestClient {
+        let client = reqwest::ClientBuilder::new()
+            .timeout(std::time::Duration::from_secs(4))
```

Comment:
nit: it might make sense to make this timeout (and others) configurable somewhere